src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -288,17 +278,103 @@ public static Sprite getMoveCursorArrowSprite()
 
     return moveCursorArrowSprite;
   }
+  
+  /**
+   * Sets up the available palettes for use.
+   * Reads in the first 6 colors on each row.
+   * Bottom colors are buildings, top are units.
+   * The key color is the last color on the top row.
+   */
+  private static void initResources()
+  {
+    if (null == mapUnitColorPalettes )
+    {
+      buildingColorPalettes = new HashMap<Color, ColorPalette>();
+      mapUnitColorPalettes = new HashMap<Color, ColorPalette>();
+      colorNames = new HashMap<Color, String>();
+      factionNames = new ArrayList<String>();
+
+      // Create a mapping of game colors to the fine-tuned colors that will be used for map sprites.
+      buildingColorPalettes.put(Color.PINK, new ColorPalette(pinkMapBuildingColors));
+      buildingColorPalettes.put(Color.CYAN, new ColorPalette(cyanMapBuildingColors));
+      buildingColorPalettes.put(Color.ORANGE, new ColorPalette(orangeMapBuildingColors));
+      buildingColorPalettes.put(PURPLE, new ColorPalette(purpleMapBuildingColors));
+
+      mapUnitColorPalettes.put(Color.PINK, new ColorPalette(pinkMapUnitColors));
+      mapUnitColorPalettes.put(Color.CYAN, new ColorPalette(cyanMapUnitColors));
+      mapUnitColorPalettes.put(Color.ORANGE, new ColorPalette(orangeMapUnitColors));
+      mapUnitColorPalettes.put(PURPLE, new ColorPalette(purpleMapUnitColors));
+
+      // Throw some color names in there for the defaults
+      // toString() is not user-friendly
+      colorNames.put(Color.PINK, "salmon");
+      colorNames.put(Color.CYAN, "cyan");
+      colorNames.put(Color.ORANGE, "citrus");
+      colorNames.put(PURPLE, "sparking");
```

Comment:
Gotta ask - where did Sparking come from? Salmon and Citrus at least make sense, but that one's out of the blue - er, purple maybe.
I think I want to do some re-branding with these color names, at any rate.

Reply:
There's an AWBW team named Purple Lightning, so I couldn't just use "purple".
"Violet" wouldn't really fit either.
"Sparking" fits with "Lightning" just fine, and kinda-sorta fits. *shrug*

Reply:
Why does "Violet" not fit? Look, some Violets:
https://duckduckgo.com/?q=violet&atb=v87-5_g&iax=images&ia=images

Also, what about Argent? It looks like that's normally more of a silver-grey color, not a deep blue. Plus I feel like that color scheme isn't balanced as well as it could be.
Lavender looks excellent, though.

Reply:
....Well, okay. "Violet" wouldn't exactly thrill me as part of a country name, but I'm okay with it if you want it lol.

Argent and Lavender were both made by other people; I simply stole with permission (and credit).
Argent's building palette's just a copy-paste with no basis, so I think it's fair to go wild and change that if desired.
Lavender was made by BlueLink; I'm sure he'd appreciate hearing your praise, hah.

Reply:
> Argent's building palette's just a copy-paste with no basis

Copy-paste from what? Also, I dunno if changing it to silver would be appreciated... why is it named Argent, anyway?
> "Violet" wouldn't exactly thrill me as part of a country name

Heh. Wait until you hear what I've got in mind for Salmon.

Reply:
> > Argent's building palette's just a copy-paste with no basis
> 
> Copy-paste from what? Also, I dunno if changing it to silver would be appreciated... why is it named Argent, anyway?

Copy-paste of the unit colors, of course.
Original: https://cdn.discordapp.com/attachments/455901857798291459/542494961484955648/unknown.png
Post with context: https://discordapp.com/channels/313453805150928906/455901857798291459/542494961484955649

> > "Violet" wouldn't exactly thrill me as part of a country name
> 
> Heh. Wait until you hear what I've got in mind for Salmon.

Oh, goody. I can't wait.
(It'd better not be Pink, that's taken already)

Reply:
Oh, the building colors are just copies of the unit colors? That explains why the units look fine, but the buildings look wonky. As for the name, Azure (which someone at source suggested) is closer than Argent to what they actually are. I might tweak the building colors if I have time.

Nope, not Pink. It's gonna be _classy_. Plus it dovetails nicely with DefendPeace, so there's that.

Reply:
Oooh, they are pretty close to Periwinkle too.

Reply:
Azure works for me.

I'm... less fine with Periwinkle.

Reply:
Robin's-Egg? Uranian? Ruddy? Argentinian? Slate? Cornflower? Steel? So many options.

Reply:
Uranian is a little out there, but palatable.
Ruddy's okay.
Slate is too.
Steel... is a little weird.
The others are kind of terrible IMO.

Reply:
Haha, the only one's I'd actually consider are Slate/Steel and maybe Uranian... maybe. Azure would also be fine. Maybe I'll roll a die or something.

Reply:
I feel like Azure fits best.